### DISCUSSION TITLE
关于进展标签

### DISCUSSION THREAD

**Discussion by yuggbh** [WatsonYao/nian_app#107] [Q&A] [answered]
我创建了一个和原有标签一样的空白标签，把空白标签删除后，原有进展上的标签使用界面正常，但该标签消失了，不管是进展页面选择还是标签管理页面

**Comment by WatsonYao** [comment]
方便邮箱发一下录屏吗。
不建议使用空白标签，会有异常情况产生。

**Reply by yuggbh** [reply]
不好意思，刚刚了解到标签批量增删之后，就又重新建了一个相同标签B，然后在批量增删中就出现了该标签且仅有一个，现在也没办法恢复，但是选择标签B在标签批量增删中 ，也同样显示标签a关联的进展。

**Reply by yuggbh** [reply]
那个我指的空白标签是没有关联任何一个进展 ，不是说其中显示空白。即我修改得到了两个标题完全相同的标签，其中一个关联的进展A， 另外一个没有关联B。然后我再删除多余标签b时 标签a也消失了。在该记本内左侧与进展可以看到标签存在，但是选择与管理标签页面没有

**Reply by WatsonYao** [reply, marked as answer]
#在该记本内左侧与进展可以看到标签存在，但是选择与管理标签页面没有
你可以把存在的进展标签重新创建一遍，然后再批量操作。
